docs/_providers/hexonet.md

```diff
@@ -0,0 +1,87 @@
+---
+name: HEXONET
+title: HEXONET Provider
+layout: default
+jsId: HEXONET
+---
+# HEXONET Provider
+
+HEXONET is a leading developer and operator of domain names and DNS platforms.  Thousands of individuals, service providers and registrars globally choose HEXONET for domains and DNS for our advanced technology, operational performance and up-time, and most importantly for unparalleled expertise in DNS. DnsControl with HEXONET's DNS marries DNS automation with an industry-leading DNS platform that supports DNSSEC, PremiumDNS via Anycast Network, and nearly all of DnsControl's listed provider features.
```

Comment:
Please limit lines to 80 cols (makes future diffs smaller).

Also... would you mind removing the sentence "Thousands ... expertise in DNS" as it seems like a marketing pitch.

Reply:
Hey @tlimoncelli,

it comes from our marketing department, I will talk to them if we can drop that sentence out.
Maybe just leaving "Thousands" out already removes that marketing pitch impression.
What you think?

Reply:
I've updated the text. Is that version ok for you?